### DISCUSSION TITLE
What code should I adjust to remove AutoFocusPlugin with `$createRangeSelection` ?

### DISCUSSION THREAD

**Discussion by Michael-indochat** [facebook/lexical#4917] [Q&A]
My page is about a form page, and the fields need to be filled in order from top to bottom, and my editor is at the bottom column, if AutoFocus is used, it will automatically slide to the bottom, but I don't want this , so I need to remove the AutoFocusPlugin inside the editor.

But at this time, after I removed AutoFocusPlugin, this error appeared when i first time to click insertMention, how can I fix it?

![image](https://github.com/sodenn/lexical-beautiful-mentions/assets/114893149/51885136-1770-4e96-9c47-c70cc311fb5c)

---
i find error problem in this func. 
how can i fix it, when my selection is null, how can I put insertNodes at the end of the entire editor content.

```typescript
  const setSelection = useCallback(() => {
    const selection = $getSelection();
    if (!selection) {
      // FIXME: if i never focus in editor, i setSelection will be undefined, how can i fix it
      $setSelection(oldSelection || $createRangeSelection());
      console.log($setSelection(oldSelection || $createRangeSelection()));
    }
    if (oldSelection) {
      setOldSelection(null);
    }
  }, [oldSelection]);
```

**Comment by matt-s-lee** [comment]
Check out this thread for a possible answer: https://github.com/facebook/lexical/issues/4226
